packages/taquito/src/wallet/register-global-constant-operation.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    BlockResponse,
+    OperationContentsAndResultRegisterGlobalConstant,
+    OpKind,
+  } from '@taquito/rpc';
+  import { Observable } from 'rxjs';
+  import { Context } from '../context';
+  import { findWithKind } from '../operations/types';
+  import { WalletOperation, OperationStatus } from './operation';
+  import { ObservableError } from './errors';
+  
+  export class RegisterGlobalConstantWalletOperation extends WalletOperation {
```

Comment:
can we add revealOperation? ref packages/taquito/src/wallet/transfer-ticket-operation.ts line 21
if the source is unrevealed will be useful that they can access the reveal op added before registerGlobalconstant op